### PR TITLE
fix input overhead on mobile

### PR DESCRIPTION
**Pull request by tarasglek**
Basically ripped out all kb logic from mobile

### CODE REVIEW

File: src/Chat/ChatBase.tsx [hunk 1]
```diff
@@ -384,7 +392,8 @@ function ChatBase({ chat }: ChatBaseProps) {
 
   const defaultProviderAlert = useMemo(() => {
     // If we are using default provider, show alert banner to notify user
-    if (showAlert) {
+    // add impossible condition so lint doesn't complete re `React Hook useMemo has an unnecessary dependency: 'settings.currentProvider'.`
```
Comment (humphd) [inline src/Chat/ChatBase.tsx:395]: Let's not do this and add a lint ignore comment

**Review by humphd — CHANGES_REQUESTED** [review]
Seems to work well for me in my limited testing.

@Amnish04, any issues you see here?

File: src/Chat/ChatBase.tsx (hunk 1 above)
Comment (tarasglek) [reply src/Chat/ChatBase.tsx]: ok dropped it for now

**Review by Amnish04 — COMMENTED** [review]
@humphd @tarasglek This looks good to me! The only catch is that people who connect keyboard with mobile would no longer be able to use the shortcuts. But that's not too common, so ig better to remove this in favour of performance.

**Pull request merged**